easybuild/easyconfigs/h/HyPhy/HyPhy-2.5.1-gompi-2019a.eb

```diff
@@ -0,0 +1,36 @@
+# This file is an EasyBuild reciPY as per https://easybuilders.github.io/easybuild/
+
+easyblock = "CMakeMake"
+
+name = 'HyPhy'
+version = '2.5.1'
+
+homepage = 'https://veg.github.io/hyphy-site/'
+description = """HyPhy (Hypothesis Testing using Phylogenies) is an open-source software package 
+ for the analysis of genetic sequences (in particular the inference of natural selection) 
+ using techniques in phylogenetics, molecular evolution, and machine learning"""
+
+toolchain = {'name': 'gompi', 'version': '2019a'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['https://github.com/veg/hyphy/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = ['c402aab5eb483c1d3566cbbd620db95cba1d86f3bb60cc7f712d255c777a45e6']
+
+builddependencies = [('CMake', '3.13.3')]
+
+buildopts = [
+    'hyphy',
+    'HYPHYMPI',
```

Comment:
buildopts should preferably (for style) be below dependencies, before sanity_check_paths.